go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/multiformats/go-multihash v0.2.3
 	github.com/onflow/atree v0.8.0-rc.5
 	github.com/onflow/cadence v1.0.0-preview.48
-	github.com/onflow/crypto v0.25.1
+	github.com/onflow/crypto v0.25.2-0.20240813163121-a1d4a1128629
```

Comment:
can we use a tagged version?

Reply:
you're right, I could tag a new crypto version

Reply:
Before I tag a new version, I'd like to also add more dependency upgrades to the crypto module: https://github.com/onflow/crypto/pull/13

Reply:
Updated to a tagged version https://github.com/onflow/flow-go/pull/6333/commits/819c2104dfd17c386d2184e836443b71296c270c